### DISCUSSION TITLE
If/else

### DISCUSSION THREAD

**Discussion by christina-de-martinez** [christina-de-martinez/babel-plugin-glowup-vibes#40] [Polls]
Choose your favorite

**Comment by aadillpickle** [comment]
iykyk: "if"
jkUnless: "else if"
nah: "else"

^tbh id go with this arrangement, the fr's are dupes, idk what what if i is (but im biased cause i proposed iykyk lol)

**Comment by Cleman25** [comment]
WhatIf
JkUnless
Nah

**Reply by Jpobi** [reply]
This but have "perchance" as an alternative "if"

**Comment by Darkoyd** [comment]
I liked the perchance idea, maybe for inline ifs?

**Comment by Fern-Ali** [comment]
if => vibeCheck
elseIf => vibeVibing
else => vibeSus

**Comment by christina-de-martinez** [comment]
Let's go with whatIfI / jkUnless / perchance -> if / else if / else
